### PR TITLE
Fixes #41 and made some other improvements

### LINKED ISSUE #41: Slay message is missing admin name
image

https://github.com/BadgerCode/tttdamagelogs/blob/b5fd4332e188b0ef1845c1458303c1638fff6994/lua/damagelogs/shared/autoslay.lua#L194-L195

This seems to have been a problem for quite some time.

### PR DESCRIPTION
**Pull request by SenseiHiraku**
I´ve added a new config option to print the amount of remaining slays of a user and improved the german translation to match up with the newest one and changed "Bericht" to "Report", because no one really says "Bericht" around here.

Made some further changes in the client panels, I´ve unchecked the "Custom Reason" check per default in the slay panel as well as autoupdating bantime of a user in the banpanel.
Fixed also a little issue regarding the amount of slays and improved some variables in the sam commands.

### CODE REVIEW

**Comment by BadgerCode** [comment]
Hey! Thanks for raising this.

I will try look over this request sometime this week!

File: lua/damagelogs/shared/lang/deutsch.lua [hunk 1]
```diff
@@ -393,5 +394,15 @@ DamagelogLang.deutsch = {
...
+    webhook_report_forgiven_or_kept = "Vergeben oder Behalten",
+    webhook_report_forgiven = "Vergeben",
+    webhook_report_kept = "Behalten",
+    webhook_report_finished_by = "Report bearbeitet von",
+    webhook_report_finished_time_taken = "Gebrauchte Zeit",
+    webhook_report_finished_conclusion = "Schlussfolgerung",
+    webhook_header_report_submitted = "Ein Report wurde eingereicht (ID %s)",
+    webhook_header_report_forgiven = "Report vergeben (ID %s)",
+    webhook_header_report_kept = "Report behalten (ID %s)",
+    webhook_header_report_finished = "Report beendet (ID %s)"
```
Comment (BadgerCode) [inline lua/damagelogs/shared/lang/deutsch.lua:407]: Thanks a lot for adding/fixing these!

File: lua/damagelogs/server/autoslay.lua [hunk 2]
```diff
@@ -500,15 +503,20 @@ hook.Add("TTTBeginRound", "Damagelog_AutoSlay", function()
...
+
                 if IsValid(v.server_ragdoll) then
                     local ply = player.GetBySteamID(v.server_ragdoll.sid)
                     if not IsValid(ply) then return end
                     ply:SetCleanRound(false)
                     ply:SetNWBool("body_found", true)
 
-                    print("ply:GetRole() " .. tostring(ply:GetRole()))
-                    print("ply:IsRole(")
-
```
Comment (BadgerCode) [inline lua/damagelogs/server/autoslay.lua:511]: Oh damn. Good spot.
This was my doing, recently.

**Review by BadgerCode — COMMENTED** [review]
Looks good! I will need to give this a quick test in-game.

File: lua/damagelogs/shared/lang/deutsch.lua (hunk 1 above)
Comment (SenseiHiraku) [reply lua/damagelogs/shared/lang/deutsch.lua:407]: Yeah np, let me know if I can help you with more translations in this or future projects 💪

File: lua/damagelogs/server/autoslay.lua [hunk 3]
```diff
@@ -12,7 +12,7 @@ if not sql.TableExists("damagelog_autoslay") then
 		ply varchar(32) NOT NULL,
 		admins tinytext NOT NULL,
 		slays SMALLINT UNSIGNED NOT NULL,
-		reason tinytext NOT NULL,
+		reason varchar(255) NOT NULL,
```
Comment (BadgerCode) [inline lua/damagelogs/server/autoslay.lua:15]: This change _should_ be fine.
It won't affect anyone who already has this table in their server database.

I'm not sure if there are any significant differences between these two types in the version of SQLite that GMod uses.
Possibly around allowed default values.

**Comment by BadgerCode** [comment]
Looks good!

#63 QA notes
* ✅ "Custom reason" not selected by default in the slay panel
* ✅ Ban time in ban panel auto updates as you change it
* ✅ Test in German- No errors/missing text
* ShowRemainingSlays
    * Off
        * ✅ No effect when a player is slain
    * On
        * ✅ Everyone gets told when a player has slays remaining
        * ✅ After all slays are done, no message
* ✅ Admin name in slays should be correct

**Comment by BadgerCode** [comment]
image

**Comment by BadgerCode** [comment]
Thanks a lot for these changes!
I'll release this now

**Pull request merged**